### DISCUSSION TITLE
奇忽局01号理事会：奇忽局社区建设理事会

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#16] [General]
根据[我和 @askender 刚才的讨论](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-569499)
成立奇忽局01号理事会：奇忽局社区建设理事会。
有意愿有时间的人可以自由加入，自由退出。在这里跟帖的人就视为加入这个理事会。
这个理事会的规则有待定义。

----
以下用于抛砖引玉，by @askender, 欢迎参与编辑，稍后会在wiki里不断完善。
定义（草稿）：当前是一个刚开启的实验：扶持构建多个去中心化内容社区，并促进高质量内容的产生、传播，并将价值更合理的回馈给创作者和参与者。
当前参与实验的有：奇忽局社区、暗问科幻社区、likecoin社区

规则&玩法（草稿）：
- 添加评论即可加入或者退出
- 每周在ch简单交流一次，采用罗伯特议事原则
  1. [Wikipedia - 罗伯特议事原则](https://zh.wikipedia.org/zh/%E7%BD%97%E4%BC%AF%E7%89%B9%E8%AE%AE%E4%BA%8B%E8%A7%84%E5%88%99)
  2. [简书 - 《罗伯特议事规则》的12条基本原则（极简版）](https://www.jianshu.com/p/bd33834dd44e)
  3. [知乎 - 罗伯特议事规则：用12原则6步法，开好一个会](https://zhuanlan.zhihu.com/p/29639932)
- 暂定大致分 底层开放协议、内容、激励机制、开源的社区（github-discussion国内访问不方便）等几个小组
- 有其他想法可以编辑规则和玩法（不要直接删除文字）
- 相关具体计划产出稍后使用git-repo来保存和更新。
- 欢迎任何人参与讨论和贡献

**Comment by Sleeper2060** [comment]
奇忽局非常有意思，🙋想加入。
本人艺术工作者，活跃在时尚圈和艺术圈，作品多以数码摄影和与3D建模相关。
Bsed在纽约，可以以美国区时间操作，有时候是欧洲时间。

**Reply by Lupingyue** [reply]
> 奇忽局非常有意思，🙋想加入。
> 本人藝術工作者，活躍在時尚圈和藝術圈，作品多以數碼攝影和與3D建模相關
> 。Bsed在紐約，可以以美國區時間操作，有時候是歐洲時間。

热烈欢迎。
1，你对于“社区建设理事会”有什么建议？
2，你是否有兴趣建设新的理事会？比如NFT策展理事会？或者其它

**Reply by Sleeper2060** [reply]
1，对于“社区建设理事会”，我希望看到白皮书或者manifesto定稿后，有针对的进行社群的建设和维护。如果仅仅是针对“去中心”这个概念，可以利用信息共享、共同协作的方式来维护社区群体。例如由理事会成员利用github建立一个成员名单，由成员自己进行登记填写。国内的共享平台要慢慢了解，国外一般用GOOGLE SHARE对文件进行共享，每个人有邮箱都可以进行登记。发展可能就由子理事会自行制定。社区理事会可以多策划活动及收集全球比较新的咨询供给成员了解，促进它们成立新的子理事会，或者说不同的项目理事会。
2，NFT策展理事会我觉得不错，这样有兴趣了解NFT或者想做NFT的创作者都可以参与进来。

**Reply by Lupingyue** [reply]
> 1，对于“社区建设理事会”，我希望看到白皮书或者manifesto定稿后，有针对的进行社群的建设和维护。如果仅仅是针对“去中心”这个概念，可以利用信息共享、共同协作的方式来维护社区群体。例如由理事会成员利用github建立一个成员名单，由成员自己进行登记填写。国内的共享平台要慢慢了解，国外一般用GOOGLE SHARE对文件进行共享，每个人有邮箱都可以进行登记。发展可能就由子理事会自行制定。社区理事会可以多策划活动及收集全球比较新的咨询供给成员了解，促进它们成立新的子理事会，或者说不同的项目理事会。
> 2，NFT策展理事会我觉得不错，这样有兴趣了解NFT或者想做NFT的创作者都可以参与进来。

1，非常有建设性的建议，听听 @askender 的意见。
2，你来新建一个”NFT策展理事会“吧，建议标题为：”奇忽局05号理事会：NFT策展理事会“。你建好之后我加入。

**Reply by Sleeper2060** [reply]
👌

**Reply by Sleeper2060** [reply]
我改一下NFT=》NFTs，意思是NFT不同形式的art work，影像，声音，电子游戏等。

**Reply by Lupingyue** [reply]
> 我改一下NFT=》NFTs，意思是NFT不同形式的art work，影像，声音，电子游戏等。

同意。

**Reply by askender** [reply]
> 1，对于“社区建设理事会”，我希望看到白皮书或者manifesto定稿后，有针对的进行社群的建设和维护。如果仅仅是针对“去中心”这个概念，可以利用信息共享、共同协作的方式来维护社区群体。例如由理事会成员利用github建立一个成员名单，由成员自己进行登记填写。国内的共享平台要慢慢了解，国外一般用GOOGLE SHARE对文件进行共享，每个人有邮箱都可以进行登记。发展可能就由子理事会自行制定。社区理事会可以多策划活动及收集全球比较新的咨询供给成员了解，促进它们成立新的子理事会，或者说不同的项目理事会。
> 2，NFT策展理事会我觉得不错，这样有兴趣了解NFT或者想做NFT的创作者都可以参与进来。

白皮书或者manifesto定稿，我觉得是挺难的，目前方向还是需要不断调整。但是可以写一个**开放式的规则和玩法**，社区来决定发展方向。    刚简单写了一下，见第一条评论

「针对“去中心”这个概念」是一个起点，最终还是要社区产生的内容来沉淀更多共识。   「可以利用信息共享、共同协作的方式来维护社区群体。」目前是这样做的，只是不想再将大话说在前面，做出来了然后再参与写白皮书吧。 当前是一个刚开启的实验：扶持构建多个去中心化内容社区，并促进高质量内容的产生

此前的部分想法记录在： https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-570489
有空我会总结一下，如果大家能参与编写白皮书就更好了。

对于2， 作为开发者，我觉得eth当前的gas较高（不知道什么时候能解决），我在尝试flow里的玩法。

**Comment by MingxuanXie** [comment]
社区建设理事会#16
平台合作社理事会#26
可持续发展理事会#18
奇宣部#28

感觉以上四个有重叠部分，都跟社区运营有关，希望能有更详细的描述划分其区别。如果一个理事会里成员跟另外一个理事会完全一样，是不是就暂时先合并管理？

看了一眼我们目前的活跃成员可能只有12位，而理事会已经有了八个😂

**Comment by Lupingyue** [comment]
> 社区建设理事会#16
> 
> 平台合作社理事会#26
> 
> 可持续发展理事会#18
> 
> 奇宣部#28
> 
> 
> 
> 感觉以上四个有重叠部分，都跟社区运营有关，希望能有更详细的描述划分其区别。如果一个理事会里成员跟另外一个理事会完全一样，是不是就暂时先合并管理？
> 
> 
> 
> 看了一眼我们目前的活跃成员可能只有12位，而理事会已经有了八个😂
> 
> 

平台合作社是李大维和邱林川老师的独立项目，我只是给他们占个位置😀
可持续发展是专注于“激励机制”的，相当于“多巴胺”和“内啡肽”的蓄水池，有必要单独成为一个主题。
奇宣部是专注于奇忽局的核心信息发布的。目前 @鹿小仙女 会志愿参与少量编辑工作。不过她账号问题可能暂时注册不了GitHub。

接下来应该不会再有大量理事会增加，除非是参与者根据自己兴趣添加。
暂时先放着，用进废退，一段时间如果活跃度太低就关闭了。
大家只关注自己有兴趣的两三个理事会就可以。

**Comment by Lupingyue** [comment]
现在已经可以从首页的「友情链接」栏目访问 @MingxuanXie  的[赛博桃花源 ](https://qihuju.wiki/peach.land/)

![IMG_2534](https://user-images.githubusercontent.com/25149468/114263641-68eaa480-9a19-11eb-9c54-5240cd887d48.PNG)

**Reply by Tz-H** [reply]
> 现在已经可以从首页的「友情链接」栏目访问 @MingxuanXie 的[赛博桃花源 ](https://qihuju.wiki/peach.land/)
> 

@MingxuanXie 的友情链接已更新，更名为“谢明炫的赛博空间”并有独立的访问页。

访问链接：[https://qihuju.wiki/docs/qihuju-links/mingxuanxie.html](https://qihuju.wiki/docs/qihuju-links/mingxuanxie.html)

编辑链接：[https://github.com/qihuju/qihuju.github.io/blob/main/docs/qihuju14.md](https://github.com/qihuju/qihuju.github.io/blob/main/docs/qihuju14.md)

**Comment by askender** [comment]
本来写了几个玩法，但不小心弄丢了。之后再陆续补上。
先写点观察吧：
- 理事会这样的名字参考的是维基百科，但也有人评价似乎比较官僚用语？我也觉得用词有点重，是否改名为兴趣组？
- 当前有9个理事会或者说兴趣组。  分别是：社区、激励、维基、游戏、NFT、合作社、农村、宣传、语音玩法9个。  明显后面6个会更热门。  我只参与了前3个。社区非常抽象不好理解似乎包含一切（稍后会具体解释），维基则非常人们需要主动投入但短期激励很少，激励机制由于采用加密货币很容易被误解。

对于热门的东西，我会开放态度的观望。当前最热门的是：NFT、声音和游戏。这三者要玩好实际上是非常重开发的。否则就是用用已有的平台了。

社区是一切的起点，解释了为什么大家要来这里。如果是因为这里有新鲜玩法和新鲜的人，热情总会过去的。留下的是具体的内容和曾经的探索。      更具体一点，留下的是： 文字、声音、图片、视频、代码、网站、链上信息等。     我想说的是社区最终凝聚的不是好奇的人，而是一些共识，当前的共识是：好奇、开源、未来玩法。   而这些背后有更底层的思考和缘起，那些才是更基本的共识。

社区的人，目前来看，都是clubhouse引流，暂时停留在微信，然后到github和discord，但实际上最终来到github和discord上的人很少。  背后的原因不是因为没有宣传，而是由于共识有些单薄导致能引发的后续行动比较少。

参与社区最简单的方式，1 在discord和github上通过文字交流，而非微信（没有thread机制），也不用过于倚重clubhouse；   2，  增加激励机制的玩法，这个我会在https://github.com/qihuju/qihuju.github.io/discussions/18 中具体实践。

**Reply by Lupingyue** [reply]
晚点细读

**Reply by askender** [reply]
具体实践1如下：    https://github.com/qihuju/qihuju.github.io/discussions/18#discussioncomment-608652   开始空投了。

**Reply by Sleeper2060** [reply]
非常同意 @askender 说的需要更多有共识的人而不是好奇的人。
接下来就是怎么思考怎么借助好奇的人帮我们找有共识的人，
个人觉得可以写个文章之类的，让他们分享出去吸引“有共识的：好奇、开源、未来玩法”。

**Reply by askender** [reply]
@Sleeper2060   都需要的， 有共识更方便协作和拓展玩法。
「怎么找到这两者」作为同行者，1 clubhouse之类用于拉新  2 在github和discord里的更深入的非线性的文字交流并记下todo-list
写文章要考虑质量和传播的问题。当下沉淀还不足，我刚在discord里开了一个  meta-talk 用于讨论，我觉得内部的ch交流得带上discord同步交流。 我提议默认周一的晚上8点以文字先开始，也要看其他人的时间。
目前前三个小组（理事会）社区、wiki、激励，可以合并沟通，其他组不太了解，可以随时乱入吧。 当前进展： 社区不用翻墙版本在开发（当前先用github和discord挺好的），激励机制也会在开发的社区中体现（已经初步测试了token的流转）、wiki是沉淀内容的地方，有空大家可以写写文章，更容易体系化。

**Comment by Sleeper2060** [comment]
整个理事会是否可以开定时长的例会互相协调一下？
例如每周开一次一个小时的会议，
用10分钟到15分钟讨论每个子理事会的事务。
现在理事会的人还不多，
好像理事会单独开会不现实。
总理事会开会还可以互相之间协调或者分享资源。

**Reply by Lupingyue** [reply]
可以在discord组织例会，好像discord的界面比较适合开例会。

**Reply by askender** [reply]
嗯，参考https://github.com/qihuju/qihuju.github.io/discussions/16#discussioncomment-630392   同意在discord 组织和发起
